src/Renderer/ThreeExtented/GlobeControls.js

```diff
@@ -651,121 +765,155 @@ function GlobeControls(camera, domElement, engine) {
-
-            state = CONTROL_STATE.DOLLY;
-
-            dollyStart.set(event.clientX - event.target.offsetLeft, event.clientY - event.target.offsetTop);
-        } else if (event.button === this.mouseButtons.PAN) {
-            if (this.enablePan === false) return;
+            if (state !== CONTROL_STATE.NONE) {
+                this.domElement.addEventListener('mousemove', _handlerMouseMove, false);
+                this.domElement.addEventListener('mouseup', _handlerMouseUp, false);
+                this.domElement.addEventListener('mouseleave', _handlerMouseUp, false);
+                this.dispatchEvent(this.startEvent);
+            }
+        });
+    };
 
-            state = CONTROL_STATE.PAN;
+    var ondblclick = function ondblclick(event) {
+        // Double click throws move camera's target with animation
+        if (!keyCtrl && !keyShift) {
+            ptScreenClick.x = event.clientX - event.target.offsetLeft;
+            ptScreenClick.y = event.clientY - event.target.offsetTop;
 
-            panStart.set(event.clientX - event.target.offsetLeft, event.clientY - event.target.offsetTop);
-        }
+            const point = getPickingPosition(ptScreenClick);
 
-        if (state !== CONTROL_STATE.NONE) {
-            this.domElement.addEventListener('mousemove', _handlerMouseMove, false);
-            this.domElement.addEventListener('mouseup', _handlerMouseUp, false);
-            this.dispatchEvent(this.startEvent);
+            if (point) {
+                animatedScale = 0.6;
+                this.setCenter(point, true);
+            }
         }
     };
 
-    var onDblClick = function onDblClick()
-    {
-        // state = CONTROL_STATE.ORBIT;
-        // scale = 0.3333;
-        // update();
-        // state = CONTROL_STATE.NONE;
-    };
-
     var onMouseUp = function onMouseUp(/* event */) {
         if (this.enabled === false) return;
 
         this.domElement.removeEventListener('mousemove', _handlerMouseMove, false);
         this.domElement.removeEventListener('mouseup', _handlerMouseUp, false);
+        this.domElement.removeEventListener('mouseleave', _handlerMouseUp, false);
         this.dispatchEvent(this.endEvent);
 
-        updateGlobeTarget.bind(this)();
-        state = CONTROL_STATE.NONE;
+        player.stop();
+
+        // Launch damping movement for :
+        //      * CONTROL_STATE.ORBIT
+        //      * CONTROL_STATE.MOVE_GLOBE
+        if (this.enableDamping) {
+            if (state === CONTROL_STATE.ORBIT && (sphericalDelta.theta > EPS || sphericalDelta.phi > EPS)) {
+                player.play(dampingOrbitalMvt).then(() => this.stopAnimation());
+            } else if (state === CONTROL_STATE.MOVE_GLOBE && lastRotation.length === 2 && !(lastRotation[1].equals(lastRotation[0]))) {
+                ctrl.qDelta.setFromUnitVectors(lastRotation[1], lastRotation[0]);
+                player.play(animationDampingMove).then(() => this.stopAnimation());
```

Comment:
it's bad name... 
I rename `reinitiateControls`, this function reinitiates somes controls's params.